delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/indexing/BlazegraphIndexingCoordinator.scala

```diff
@@ -0,0 +1,167 @@
+import ch.epfl.bluebrain.nexus.delta.plugins.blazegraph.client.{BlazegraphClient, SparqlClientError, SparqlWriteQuery}
+import ch.epfl.bluebrain.nexus.delta.plugins.blazegraph.model.BlazegraphView.IndexingBlazegraphView
+import ch.epfl.bluebrain.nexus.delta.plugins.blazegraph.model.BlazegraphViewsConfig
+import ch.epfl.bluebrain.nexus.delta.rdf.RdfError.InvalidIri
+import ch.epfl.bluebrain.nexus.delta.rdf.Triple.Triple
+import ch.epfl.bluebrain.nexus.delta.rdf.Vocabulary.{nxv, rdf}
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.ExpandedJsonLd
+import ch.epfl.bluebrain.nexus.delta.sdk.eventlog.GlobalEventLog
+import ch.epfl.bluebrain.nexus.delta.sdk.implicits._
+import ch.epfl.bluebrain.nexus.delta.sdk.indexing.IndexingStreamCoordinator
+import ch.epfl.bluebrain.nexus.delta.sdk.model.IdSegment.IriSegment
+import ch.epfl.bluebrain.nexus.delta.sdk.model.{BaseUri, ResourceF}
+import ch.epfl.bluebrain.nexus.sourcing.projections.ProjectionId.ViewProjectionId
+import ch.epfl.bluebrain.nexus.sourcing.projections.ProjectionStream.{ChunkStreamOps, SimpleStreamOps}
+import ch.epfl.bluebrain.nexus.sourcing.projections.stream.StreamSupervisor
+import ch.epfl.bluebrain.nexus.sourcing.projections.{Projection, ProjectionId, ProjectionProgress}
+import com.typesafe.scalalogging.Logger
+import fs2.Stream
+import monix.bio.Task
+import monix.execution.Scheduler
+import retry.CatsEffect._
+import retry.syntax.all._
+
+object BlazegraphIndexingCoordinator {
+
+  private val logger: Logger = Logger[BlazegraphIndexingCoordinator.type]
+
+  def apply(
+      views: BlazegraphViews,
+      eventLog: GlobalEventLog[ResourceF[ExpandedJsonLd]],
+      client: BlazegraphClient,
+      projection: Projection[Unit],
+      config: BlazegraphViewsConfig
+  )(implicit
+      as: ActorSystem[Nothing],
+      scheduler: Scheduler,
+      base: BaseUri
+  ): Task[IndexingStreamCoordinator[ResourceF[IndexingBlazegraphView], Unit]] = {
+    def idF(viewRes: ResourceF[IndexingBlazegraphView]): ViewProjectionId = ViewProjectionId(
+      s"blazegraph-indexer-${viewRes.value.project}-${viewRes.value.id}"
+    )
+
+    def buildStream(
+        viewRes: ResourceF[IndexingBlazegraphView],
+        initialProgress: ProjectionProgress[Unit]
+    ): Task[Stream[Task, Unit]] = {
+      implicit val projectionId: ProjectionId = idF(viewRes)
+      val view                                = viewRes.value
+      val namespace                           = s"${config.client.indexPrefix}_${view.project}_${view.uuid}_${viewRes.rev}"
+      val retryStrategy                       = RetryStrategy[SparqlClientError](
+        config.client.retry,
+        {
+          case WrappedHttpClientError(_) => true
+          case _                         => false
+        },
+        logError(logger, "blazegraph client")
+      )
+      import retryStrategy.{errorHandler, policy, retryWhen}
+      for {
+        _     <- client
+                   .createNamespace(namespace, Map.empty)
```

Comment:
I believe this should not be Map.empty. In v1.4.xx we had something like:
```scala
private val properties: Map[String, String] = {
      val props = new Properties()
      props.load(getClass.getResourceAsStream("/blazegraph/index.properties"))
      props.asScala.toMap
    }
```

And the index.properties file had the following content:
```
com.bigdata.rdf.store.AbstractTripleStore.textIndex=true
com.bigdata.rdf.store.AbstractTripleStore.axiomsClass=com.bigdata.rdf.axioms.NoAxioms
com.bigdata.rdf.sail.isolatableIndices=false
com.bigdata.rdf.sail.truthMaintenance=false
com.bigdata.rdf.store.AbstractTripleStore.justify=false
com.bigdata.namespace.test-ns.spo.com.bigdata.btree.BTree.branchingFactor=1024
com.bigdata.rdf.store.AbstractTripleStore.quads=true
com.bigdata.namespace.test-ns.lex.com.bigdata.btree.BTree.branchingFactor=400
com.bigdata.rdf.store.AbstractTripleStore.geoSpatial=false
com.bigdata.rdf.store.AbstractTripleStore.statementIdentifiers=false
```